doc/reference/misc.md

```diff
@@ -2058,15 +4231,19 @@ Please document me!
 :::
 
 ### equal-shared?
-::: tip usage
-```
-(equal-shared? ...)
-```
-:::
+``` scheme
+(equal-shared? a b) -> boolean
 
-Please document me!
+  a, b := structures to check
+```
 
+Checks whether *a* and *b*, two potentially recursive, cyclic or otherwise
+infinite shared structures, e.g. trees or graphs, are equal.
 
+::: warning Deprecation note:
+Gambit 4.9.1 (released 2018-11-16) added similar support for handling shared
```

Comment:
it's actually in Gambit v4.9.3, sorry for the disinformation on irc.

Reply:
Alright, done.